frontend/src/components/CommentVotes.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { fetchUpvotesForComment, fetchDownvotesForComment } from '../store/actions/comments';
```

Comment:
add aliases, beacues later on you will end up i relative import hell `../../../` :)

Reply:
What exactly do you mean?

I found this: https://decembersoft.com/posts/say-goodbye-to-relative-paths-in-typescript-imports/

I will make a separate PR for that because this may break something on the way.